toolchain/driver/testdata/error_codegen_objcode_target_triple.carbon

```diff
@@ -0,0 +1,8 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{not} %{carbon} dump assembly --target_triple=x86_687-unknown-linux-gnu %s | %{FileCheck-strict}
```

Comment:
This is testing `assembly` mode not `objcode` mode, but based on the filename I think you meant for this to be an `objcode` test. You'll also need to provide an output file name or this test will be the same as `error_codegen_objcode_output_file_with_triple.carbon`.